cypress/integration/product/create_bill_of_material.js

```diff
@@ -1,11 +1,10 @@
 import { Product, ProductCategory } from '../../support/utils/product';
-import { BillOfMaterial } from '../../support/utils/billOfMaterial';
+import { BillOfMaterial, BillOfMaterialLine } from '../../support/utils/billOfMaterial';
 import { humanReadableNow } from '../../support/utils/utils';
 
-describe('Create Product', function() {
+describe('Create Product and BOM', function() {
   const date = humanReadableNow();
   const productName = `ProductName ${date}`;
```

Comment:
If we are here, can we update it to use the new fixture approach we decided on?

Reply:
I'd rather do it in the special purpose task for that: https://github.com/metasfresh/metasfresh-e2e/issues/350